/* $Id$
 * Simple Chat
 * Copyright Â© 2008 IMPOMEZIA (http://impomezia.net.ru)
 */

#ifndef WELCOMEDIALOG_H_
#define WELCOMEDIALOG_H_

#include <QDialog>

#include "ui_welcomedialog.h"

class SChatWindow;

class WelcomeDialog : public QDialog, public Ui::WelcomeDialog
{
  Q_OBJECT

public:
  WelcomeDialog(const QString &nick, const QString &name, quint8 sex, QWidget *parent = 0);
  bool hideWelcome();
  QString fullName();
  QString nick();
  quint8 sex();
  void setHideWelcome(bool w);
};

#endif /*WELCOMEDIALOG_H_*/
